fieldservice_recurring/models/fsm_frequency.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (C) 2019 - TODAY, Brian McMaster, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from dateutil.rrule import MO, TU, WE, TH, FR, SA, SU
+from dateutil.rrule import YEARLY, MONTHLY, WEEKLY, DAILY
+from dateutil.rrule import rrule
+
+from odoo import fields, models, api, _
+from odoo.exceptions import UserError
+
+
+WEEKDAYS = {
+    'mo': MO,
+    'tu': TU,
+    'we': WE,
+    'th': TH,
+    'fr': FR,
+    'sa': SA,
+    'su': SU
+}
+
+FREQUENCIES = {
+    'yearly': YEARLY,
+    'monthly': MONTHLY,
+    'weekly': WEEKLY,
+    'daily': DAILY,
+}
+
+
+class FSMFrequency(models.Model):
+    _name = 'fsm.frequency'
+    _description = 'Frequency Rule for Field Service Orders'
+    _inherit = ['mail.thread']
+
+    name = fields.Char('Name', required=True)
+    active = fields.Boolean(default=True)
+    interval = fields.Integer(
+        string='Repeat Every', help="The number of intervals between events",
+        default=1, required=True, track_visibility='onchange')
+    interval_type = fields.Selection([
```

Comment:
```suggestion
    interval_type = fields.Selection(FREQUENCIES,
```

Can you use the variable defined above?
Like here:
https://github.com/OCA/field-service/blob/12.0/fieldservice_stock/models/fsm_order.py#L43

Reply:
Done [https://github.com/OCA/field-service/pull/179/commits/709bcbd9507ea0eca154731a3babd78a89918429](https://github.com/OCA/field-service/pull/179/commits/709bcbd9507ea0eca154731a3babd78a89918429)